demo/index.html

```diff
@@ -45,8 +48,27 @@
     <div>Auto Validation</div>
 
     <paper-input-container auto-validate>
-      <label>letters only</label>
-      <input pattern="[a-zA-Z]*">
+      <label>only type letters</label>
+      <input is="iron-input" pattern="[a-zA-Z]*">
+      <paper-input-error class="add-on">letters only!</paper-input-error>
```

Comment:
We should absolutely have pre-composed elements for ease of use. For this element I was thinking about giving the option of composing your own input element if you need more flexibility, and also providing pre-composed elements for convenience. I have not yet made the convenience elements :)

The API surface for `input` is large, so it seems impractical to mirror it in entirety if we only provide a pre-composed element.